### PR TITLE
[Tizen.platform] Add face landmarks example

### PR DESCRIPTION
**Pull request by songgot**
-nnstreamer example application for face landmarks on tizen platform
-face_landmark.tflite from mediapipe is used

Signed-off-by: Hyunil <hyunil46.park@samsung.com>

### CODE REVIEW

File: Tizen.platform/Tizen_IoT_face_landmark/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+#cd opt
+#rpm -Uvh --force --nodeps tizen-iot-face-landmark-app-0.0.1-0.armv7l.rpm
+```
+To run applicaton
+```bash
+#export ELM_ENGINE=wayland_egl
+#export XDG_RUNTIME_DIR=/run/
+#export ELM_PROFILE="common"
+
+#tizen_iot_face_landmark
```
Comment (myungjoo) [inline Tizen.platform/Tizen_IoT_face_landmark/README.md:55]: Please draw pipeline in README.md (as .png ?)

Comment (myungjoo) [reply Tizen.platform/Tizen_IoT_face_landmark/README.md:55]: And add a (small size!) .webp showing how it actually works.

Comment (songgot) [reply Tizen.platform/Tizen_IoT_face_landmark/README.md:55]: okay. but inference accurancy is not good.

Comment (myungjoo) [reply Tizen.platform/Tizen_IoT_face_landmark/README.md:55]: That's usually ok :)

File: Tizen.platform/Tizen_IoT_face_landmark/src/main.c [hunk 2]
```diff
@@ -0,0 +1,557 @@
+/**
+ * @file main.c
+ * @date 8 April 2022
+ * @brief nnstreamer example application for face landmark on tizen platform
+ * @see  https://github.com/nnsuite/nnstreamer
+ * @author Hyunil Park <hyunil46.park@samsung.com>
+ * @bug No known bugs
+ */
+
+/*
```
Comment (gichan-jang) [inline Tizen.platform/Tizen_IoT_face_landmark/src/main.c:10]: Doxygen or multi line comments should begin with /**.
Please modify other lines also.

File: Tizen.platform/Tizen_IoT_face_landmark/src/main.c [hunk 3]
```diff
@@ -0,0 +1,557 @@
...
+		elm_win_fullscreen_set(win, EINA_TRUE);
+		elm_win_autodel_set(win, EINA_TRUE);
+		elm_win_alpha_set(win, EINA_TRUE);
+		elm_win_activate(win);
+		evas_object_show(win);
+	}
+	return win;
+}
+
+/*
```
Comment (gichan-jang) [inline Tizen.platform/Tizen_IoT_face_landmark/src/main.c:112]:
```suggestion
/**
```

Comment (songgot) [reply Tizen.platform/Tizen_IoT_face_landmark/src/main.c]: thank you

File: Tizen.platform/Tizen_IoT_face_landmark/src/main.c (hunk 2 above)
Comment (songgot) [reply Tizen.platform/Tizen_IoT_face_landmark/src/main.c]: okay

**Review by myungjoo — APPROVED** [review]
LGTM!

**Comment by myungjoo** [comment]
Please add the webp to main nnstreamer.git's README.md

**Pull request merged**